### ISSUE TITLE
Failed to install

### ISSUE THREAD

**Issue by yujunz** [Snaipe/dot-git#1]
Got an error in the following step

```
➜  ~ dot checkout -f --
fatal: Not a git repository: '/Users/yujunz/.config/dotgit/repo'
```

I think it is probably because it is a bare repo and `repo` is not available.

**Comment by Snaipe** [comment]
The profile management added a symlink to refer to the "current" profile, which exists at `~/.config/dotgit/repo`. It normally is versioned, but since we're creating it with the `checkout -f --`, the command is broken until then.

I obviously forgot to take that into account in the installation instructions.

To fix this, just call `ln -s . ~/.config/dotgit/repo`, and resume at the checkout step.

**Comment by Snaipe** [comment]
Fixed by 15173e7.
